docs/tutorials/rust.md

```diff
@@ -0,0 +1,152 @@
+# Tutorial: Develop a Rust package using `pixi`
+
+In this tutorial, we will show you how to develop a Rust package using `pixi`.
+The tutorial is written to be executed from top to bottom, missing steps might result in errors.
+
+The audience for this tutorial is developers who are familiar with Rust and `cargo` and how are interested to try pixi for their development workflow.
+
+!!! note ""
+    If you're new to pixi, you can check out the [basic usage](../basic_usage.md) guide.
+    This will teach you the basics of pixi project within 3 minutes.
+
+## Prerequisites
+
+- You need to have `pixi` installed. If you haven't installed it yet, you can follow the instructions in the [installation guide](../index.md).
+  The crux of this tutorial is to show you only need pixi!
+
+## Create a pixi project.
+
+```shell
+pixi init my_rust_project
+cd my_rust_project
+```
+
+It should have created a directory structure like this:
+
+```shell
+my_rust_project
+├── .gitattributes
+├── .gitignore
+└── pixi.toml
+```
+
+The `pixi.toml` file is the manifest file for your project. It should look like this:
+
+```toml title="pixi.toml"
+[project]
+name = "my_rust_project"
+version = "0.1.0"
+description = "Add a short description here"
+authors = ["User Name <user.name@email.url>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
```

Comment:
I would add a comment that platforms will differ depends on your machine ( to not create confusion )